Core/clim-core/presentation-defs.lisp

```diff
@@ -548,7 +548,7 @@ a specific type."))
                        (gethash name history-table)
                        history-object))
                history-object))
-            (nil
+            ((nil)
              nil)
```

Comment:
thank you. You are right. I've removed the comment because it is how a conforming CL implementation should behave (that is, if we look for a symbol NIL it should be enclosed in parenthesis).